gui/src/app/Scripting/Runtime/load_args.py

```diff
@@ -0,0 +1,27 @@
+def rename_sampling_options(k):
+    """
+    convert between names used in
+    Stan-Playground and CmdStanPy
+    """
+
+    if k == "init_radius":
+        return "inits"
+    if k == "num_warmup":
+        return "iter_warmup"
+    if k == "num_samples":
+        return "iter_sampling"
+    if k == "num_chains":
+        return "chains"
+    if k == "seed":
+        return "seed"
```

Comment:
I feel like I want this to be a dictionary, but I think I actually just want Python to be Typescript.